app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Sorry, Please check this language settings at file https://github.com/zgunz42/hot-movie/blob/8af7df778ce4491921236d22a210195f5d3091ad/app/src/main/res/values/arrays.xml#L24-L32 you need to add your language code and name so it can displayed on language section on settings activity https://github.com/zgunz42/hot-movie/blob/master/app/src/main/java/com/kangmicin/hotmovie/ui/setting/SettingsActivity.kt

Reply:
Thank you for your reply I just make a quick fix in the code to insert the language in the setting file :D